src/pages/UserProfilePage.tsx

```diff
@@ -1,8 +1,9 @@
-import {ChangeEvent, FormEvent, useEffect, useState} from 'react';
+import React, {ChangeEvent, FormEvent, useEffect, useState} from 'react';
```

Comment:
[We shouldn't need to import React anymore](https://www.reddit.com/r/reactjs/comments/umj534/do_i_need_import_react_from_react/), what version do you have installed locally?

Reply:
i'll play around with this for my next frontend PR